metric-prometheus/src/test/scala/com/evolutiongaming/metric/prometheus/PrometheusMetricSpec.scala

```diff
@@ -0,0 +1,57 @@
+package com.evolutiongaming.metric.prometheus
+
+import io.prometheus.client.CollectorRegistry
+import org.scalatest._
+
+class PrometheusMetricSpec extends fixture.FlatSpec with Matchers {
+
+  override type FixtureParam = PrometheusMetric
+
+  override def withFixture(test: OneArgTest): Outcome = {
+    test(new PrometheusMetric(new CollectorRegistry()))
+  }
+
+  it should "report recorded metrics" in { metrics =>
+    val g = metrics.gauge("test", "provides test gauge", ())
```

Comment:
Not really like api without labels, as always should provide some type, which in this case is Unit which has LabelGen instance. Probably method overload would do, but that will add even more boiler plate to Metrics trait. Mby some of the boilerplate can be generated using macros.